doc/standard_library.md

```diff
@@ -0,0 +1,214 @@
+# Ruby Standard Library
+
+The Ruby Standard Library is a large collection of classes and modules you can
+require in your code to gain additional features.
+
+Below is an overview of the libraries and extensions, followed by a brief description
+of each.
+
+## Libraries
```

Comment:
Shouldn't these headings `###` as well as under Default/Bundled gems?

Reply:
I misread the original headings. I've restored them now.